### DISCUSSION TITLE
How to use nvm

### DISCUSSION THREAD

**Discussion by FunnyGhost** [actions/virtual-environments#4706] [Installed software]
Hello!

On the macos-latest image, nvm is listed as an already installed utility and it already has some node versions cached. When trying to use it though, Azure PIpelines says 'nvm command not found'. Is it supposed to not be used?

Thanks!

**Comment by dmitry-shibanov** [comment]
Hello @FunnyGhost. Could you please try to use this snippet:
```
- bash: |
   . $HOME/.nvm/nvm.sh > /dev/null 2>&1 || true
   nvm ls
   nvm use v14
```

**Reply by FunnyGhost** [reply]
Thanks! Ended up doing something similar:

```
  - script: |
       export NVM_DIR="$HOME/.nvm"
       [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
       nvm use 14
       npm ci
```

**Reply by Abraham-Flutterwave** [reply]
is this still the case ?

**Reply by vpolikarpov-akvelon** [reply]
Hi, @Abraham-Flutterwave. Could you elaborate your question a bit?
